src/rpc/blockchain.cpp

```diff
@@ -983,6 +939,117 @@ UniValue gettxout(const JSONRPCRequest& request)
     return ret;
 }
 
+UniValue getutxoindex(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() < 2 || request.params.size() > 4)
+        throw std::runtime_error(
+            "getutxoindex ( minconf [\"address\",...] count from )\n"
+            "\nReturns a list of unspent transaction outputs by address (or script).\n"
+            "The list is ordered by confirmations in descending order.\n"
+            "Note that passing minconf=0 will include the mempool.\n"
+            "\nTo use this function, you must start bitcoin with the -txoutindex parameter.\n"
+            "\nArguments:\n"
+            "1. minconf          (numeric) Minimum confirmations\n"
+            "2. \"addresses\"    (string) A json array of bitcoin addresses (or scripts)\n"
+            "    [\n"
+            "      \"address\"   (string) bitcoin address (or script)\n"
```

Comment:
The address abstraction ends before/when coins hit the UTXO set. UTXOs do not have and are not associated to addresses.

Rename this to `"scripts"`, and don't support specifying as addresses.